code/modules/food_and_drinks/food/snacks_meat.dm

```diff
@@ -188,6 +188,14 @@
 	new /mob/living/carbon/monkey(get_turf(src))
 	qdel(src)
 
+/obj/item/weapon/reagent_containers/food/snacks/monkeycube/gorilla
+	name = "gorilla cube"
+
+/obj/item/weapon/reagent_containers/food/snacks/monkeycube/gorilla/Expand()
```

Comment:
Bad copypasta. Make the type it spawns a variable, this way you can do a single `Expand()` with `new monkey_type...` instead of two procs doing the exact same thing.